src/utils/writeToDisk.ts

```diff
@@ -38,7 +40,7 @@ export function writeToDisk({
       ) {
         fs.ensureFileSync(outputFilePath);
         fs.writeJSONSync(outputFilePath, otherLanguageLocaleFile.data, {
-          spaces: 2,
+          spaces: spaces,
```

Comment:
Can you change it so it will become `spaces,` instead of `spaces: spaces,`

Reply:
Done